edx_arch_experiments/datadog_monitoring/signals/handlers.py

```diff
@@ -5,13 +5,43 @@
 
 from celery.signals import worker_process_init
 from django.dispatch import receiver
+from edx_django_utils.monitoring.signals import (
+    monitoring_support_process_exception,
+    monitoring_support_process_request,
+    monitoring_support_process_response,
+)
 
 from edx_arch_experiments.datadog_monitoring.code_owner.datadog import CeleryCodeOwnerSpanProcessor
+from edx_arch_experiments.datadog_monitoring.code_owner.utils import set_code_owner_attribute
 
 log = logging.getLogger(__name__)
 
 
-@receiver(worker_process_init)
+@receiver(monitoring_support_process_response, dispatch_uid=f"datadog_monitoring_support_process_response")
+def datadog_monitoring_support_process_response(sender, **kwargs):
+    """
+    Adds datadog monitoring at monitoring process response time.
+    """
+    if 'request' in kwargs:
+        set_code_owner_attribute(kwargs['request'])
```

Comment:
It's probably worth just documenting the provided kwargs in edx-django-utils so we can put them directly in the receiver function's signature.

Reply:
1. I added directly to the signature, but I left the defensive code, since this is for monitoring. Thoughts?
2. Can you explain where you want to see the kwargs documented? Do you mean here https://github.com/openedx/edx-django-utils/blob/master/edx_django_utils/monitoring/README.rst#monitoring-support-middleware-and-monitoring-plugins? Other? Do you think this is an important change, or is it close enough to simply allow someone to search for the implementation?

Reply:
I don't think the defensive code is needed, since e-d-u is using send_robust.

I'd say document the kwargs where ever you've got other documentation -- either in the rST there or in docstrings. In my opinion it would be best for it to be as close to the code as possible, so that changes to the code are more likely to involve changes to docs/comments, but rST is probably fine too, since that's already where you describe the role and usage of the signals.